app/routes/_landing.($lang).docs.$/route.tsx

```diff
@@ -0,0 +1,69 @@
+import { LoaderFunction, json } from "@remix-run/node"
+import { useLoaderData } from "@remix-run/react"
+import invariant from "tiny-invariant"
+import NotFound404 from "~/components/NotFound404"
+import Remark from "~/components/Remark"
+import { initCmsClient } from "~/models/cms/cms.server"
+import { documentation } from "~/models/cms/sdk"
+import { getClientEnv } from "~/utils/environment.server"
+
+type LoaderData = {
+  data: documentation[]
+}
+
+export const loader: LoaderFunction = async ({ params }) => {
```

Comment:
I see this part is being repeated and duplicated in (three?) different routes. Let's move the logic out into a function and use it instead of duplicating the code.

Reply:
The logic is kinda similar for both docs route, but there're some small details that are different. You can see one has different error checks, one has a slug filter, and also the way one looks for dynamic params is different so I don't think this warrants to reuse the loader.

Reply:
Sounds good..